content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -691,9 +691,6 @@ Each feature gate is designed for enabling/disabling a specific feature:
 - `SeccompDefault`: Enables the use of `RuntimeDefault` as the default seccomp profile
   for all workloads.
   The seccomp profile is specified in the `securityContext` of a Pod and/or a Container.
-- `SELinuxMountReadWriteOncePod`: Allows kubelet to mount volumes for a Pod directly with the
-  right SELinux label instead of applying the SELinux label recursively on every file on the
-  volume.
```

Comment:
+1 to @jihoon-seo suggestions although we should keep line 694-696 and remove the 688-690 and we should add this property also `Speed up container startup by mounting volumes with the correct
  SELinux label instead of changing each file on the volumes recursively`
@jihoon-seo wdyt?

Reply:
+1 to suggestion by @jihoon-seo . I would also recommend changing the text to the below:

```
`SELinuxMountReadWriteOncePod`: Speeds up container startup by allowing kubelet to mount volumes for a Pod directly with the correct SELinux label instead of changing each file on the volumes recursively. The initial implementation
  focused on ReadWriteOncePod volumes.
```

Reply:
If we moved the lines to L694-696 the alphabetical order would be correct but not the ASCII one.
Paying attention to L378 vs L380, you'll find another example of what I mean: APIServerTracing before AdvancedAuditing.

Reply:
AFAIK, ASCII order would be maintained too because E would come before e with the suggestion effected.